Index: stage2.1/rootfs/etc/network/interfaces
===================================================================
--- stage2.1.orig/rootfs/etc/network/interfaces	2019-01-12 03:57:20.313238686 +0100
+++ stage2.1/rootfs/etc/network/interfaces	2019-01-15 18:26:43.558775829 +0100
@@ -3,5 +3,19 @@
 # Please note that this file is written to be used with dhcpcd
 # For static IP, consult /etc/dhcpcd.conf and 'man dhcpcd.conf'
 
+# localhost
+auto lo
+iface lo inet loopback
+
+# ethernet
+auto eth0
+iface eth0 inet dhcp
+
+# WLAN interface
+allow-hotplug wlan0
+iface wlan0 inet static
+address 192.168.42.1
+netmask 255.255.255.0
+
 # Include files from /etc/network/interfaces.d:
 source-directory /etc/network/interfaces.d
