--- a/rootfs/etc/dnsmasq.conf
+++ b/rootfs/etc/dnsmasq.conf
@@ -668,3 +668,15 @@
 # Delays sending DHCPOFFER and proxydhcp replies for at least the specified number of seconds.
 dhcp-mac=set:client_is_a_pi,B8:27:EB:*:*:*
 dhcp-reply-delay=tag:client_is_a_pi,2
+
+# activate DHCP server for wlan0 iface
+interface=wlan0
+
+# disable dhcpd server for wired LAN
+no-dhcp-interface=eth0
+
+# IPv4 address range and lease time
+dhcp-range=192.168.42.100,192.168.42.150,24h
+
+# DNS
+dhcp-option=option:dns-server,192.168.42.1
