--- /dev/null
+++ b/rootfs/home/pi/activate-wlan.sh
@@ -0,0 +1,8 @@
+#!/bin/bash
+# activate a local WLAN
+#
+# usage: activate-wlan ESSID PASSWORD
+
+/usr/bin/wpa_passphrase "$1" "$2" | sudo tee -a /etc/wpa_supplicant/wpa_supplicant.conf
+sudo /sbin/wpa_cli reconfigure
+sudo service dnsmasq restart
--- /dev/null
+++ b/rootfs/home/pi/list-wlans.sh
@@ -0,0 +1,5 @@
+#!/bin/bash
+# List all wireless networks locally available
+
+echo "Available wireless networks:"
+sudo /sbin/iwlist scan | grep ESSID | cut -d ':' -f 2- | uniq
