--- a/rootfs/etc/network/interfaces
+++ b/rootfs/etc/network/interfaces
@@ -3,5 +3,42 @@
 # Please note that this file is written to be used with dhcpcd
 # For static IP, consult /etc/dhcpcd.conf and 'man dhcpcd.conf'
 
+# localhost
+auto lo
+iface lo inet loopback
+
+# ethernet
+auto eth0
+iface eth0 inet dhcp
+
+# WLAN interface
+allow-hotplug wlan0
+iface wlan0 inet static
+address 192.168.42.1
+netmask 255.255.255.0
+
+allow-hotplug wlan1
+iface wlan1 inet dhcp
+    wpa-conf /etc/wpa_supplicant/wpa_supplicant.conf
+
+# delete existing firewall rules
+up /sbin/iptables -F
+up /sbin/iptables -X
+up /sbin/iptables -t nat -F
+
+# allow loopback traffic
+up /sbin/iptables -A INPUT -i lo -j ACCEPT
+up /sbin/iptables -A OUTPUT -o lo -j ACCEPT
+
+# enable NAT and masquerading
+up /sbin/iptables -A FORWARD -o wlan1 -i wlan0 -m conntrack --ctstate NEW -j ACCEPT
+up /sbin/iptables -A FORWARD -m conntrack --ctstate ESTABLISHED,RELATED -j ACCEPT
+up /sbin/iptables -t nat -A POSTROUTING -o wlan1 -j MASQUERADE
+
+# activate IP forwarding
+up sysctl -w net.ipv4.ip_forward=1
+up sysctl -w net.ipv6.conf.all.forwarding=1
+
+
 # Include files from /etc/network/interfaces.d:
 source-directory /etc/network/interfaces.d
