--- a/rootfs/etc/network/interfaces
+++ b/rootfs/etc/network/interfaces
@@ -3,5 +3,23 @@
 # Please note that this file is written to be used with dhcpcd
 # For static IP, consult /etc/dhcpcd.conf and 'man dhcpcd.conf'
 
+# localhost
+auto lo
+iface lo inet loopback
+
+# ethernet
+auto eth0
+iface eth0 inet dhcp
+
+# WLAN interface
+allow-hotplug wlan0
+iface wlan0 inet static
+address 192.168.42.1
+netmask 255.255.255.0
+
+allow-hotplug wlan1
+iface wlan1 inet dhcp
+    wpa-conf /etc/wpa_supplicant/wpa_supplicant.conf
+
 # Include files from /etc/network/interfaces.d:
 source-directory /etc/network/interfaces.d
